### DISCUSSION TITLE
Where to have discussions?

### DISCUSSION THREAD

**Discussion by timrae** [spotify2tidal/spotify_to_tidal#45] [General]
Now that it's not just me working on this project, it would be good to designate a primary place for discussions. Personally I would suggest having most discussions in public here on github, but @RobinHirst11 suggested using Discord and @joshrmcdaniel also expressed a desire for a more private place, so am open to hear what you guys think

**Comment by som-cs** [comment]
on github.

**Comment by RobinHirst11** [comment]
hm.. i guess github works....
